endoreg_db/views/video_media_views.py

```
@@ -0,0 +1,350 @@
+                status=status.HTTP_500_INTERNAL_SERVER_ERROR
+            )
+
+
+class VideoApplyMaskView(APIView):
+    """
+    POST /api/video-apply-mask/{id}/ - Apply mask to video using streaming processing
+    """
+    permission_classes = [EnvironmentAwarePermission]
+    
+    def post(self, request, pk):
+        video = get_object_or_404(VideoFile, pk=pk)
```

Ruff (0.12.2)

Local variable `video` is assigned to but never used

Remove assignment to unused variable `video`

(F841)